tests/testthat/test-fotmob.R

```diff
@@ -4,15 +4,17 @@ context("Testing fotmob functions")
 test_that("fotmob_get_matches_by_date() works", {
   testthat::skip_on_cran()
 
+  expected_matches_by_date_cols <- c("away_id", "away_name", "away_score", "ccode", "home_id", "home_name", "home_score", "id", "match_id", "match_league_id")
+
   td <- Sys.Date()
-  results <- fotmob_get_matches_by_date(date = as.character(c(td-7, td-8)))
+  results <- fotmob_get_matches_by_date(date = as.character(td - c(7, 8)))
   expect_gt(nrow(results), 0)
   ## There have been issues where the columns are not in the same exact order depending on the day, so rely on sort
-  expect_true(
-    all(
-      c("away_id", "away_name", "away_score", "ccode", "home_id", "home_name", "home_score", "id", "match_id", "match_league_id") %in% colnames(results)
-    )
-  )
+  expect_true(all(expected_matches_by_date_cols %in% colnames(results)))
+
+  future_results <- fotmob_get_matches_by_date(date = as.character(td + c(1, 2)))
```

Comment:
should have added a test with the prior PR fixing `fotmob_get_matches_by_date` for future dates